document/js-api/index.bs

```diff
@@ -753,11 +753,19 @@ which can be simultaneously referenced by multiple {{Instance}} objects. Each
     1. Otherwise, throw a {{TypeError}} exception.
 </div>
 
+<div algorithm>
+    The algorithm <dfn>DefaultGlobalValue</dfn>(|valuetype|) performs the following steps:
```

Comment:
Nit: I would probably just call this DefaultValue, since it doesn't have any dependency on globals and might potentially be useful in other contexts.